salt/orchestrate/aws/app_elb.sls

```diff
@@ -40,18 +38,20 @@ create_elb_for_{{ app_name }}_{{ ENVIRONMENT }}:
           enabled: True
           timeout: 300
     - cnames:
+    {% for zone_name, domain in zip(zone_names, domains) %}
+    {% if zone_name.strip('.') in domain %}
         - name: {{ purpose.domain }}
```

Comment:
This should be using the `domain` variable from the `zip` function call, rather than `purpose.domain`